src/__tests__/waitForElement.test.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+/* eslint-disable react/no-multi-comp */
```

Comment:
Missing test case, where you check for rejected Promise when element is not found in timeout.

Reply:
um, is this what you mean? https://github.com/callstack/react-native-testing-library/pull/20/files#diff-1c9876d94966fa35d3d6b0ccf7b9a8fbR51

Reply:
Yup C: